src/Microsoft.DotNet.Darc/Darc/Models/PopUps/AddSubscriptionPopUp.cs

```diff
@@ -1,31 +1,22 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using Microsoft.DotNet.Maestro.Client.Models;
-using Microsoft.Extensions.Logging;
 using System;
 using System.Collections.Generic;
-using System.Collections.Immutable;
 using System.Collections.ObjectModel;
 using System.Linq;
+using Microsoft.DotNet.Maestro.Client.Models;
+using Microsoft.Extensions.Logging;
 using YamlDotNet.Serialization;
 
 namespace Microsoft.DotNet.Darc.Models.PopUps;
 
-public class AddSubscriptionPopUp : EditorPopUp
+public class AddSubscriptionPopUp : SubscriptionPopUp
```

Comment:
This naming is weird (albeit I see this was already there). `Add` verb makes an impression that this is a method, not a class.

Reply:
There's a whole suite of classes called `AddSubscription` from command line options, to operations, to controller endpoints so I would keep it in line as it's very easy to navigate around the codebase.

Reply:
Ah, I see now it's "Add Subscription" pop-up, not add subscription pop up. Misunderstanding on my side.